tvision/classes/wingr/wingrscr.cc

```diff
@@ -734,8 +734,13 @@ int TScreenWinGr::System( const char * command
   }
 
   if ( pidChild )                      // Leave in the background
-  { return( *pidChild= (pid_t)pi.hProcess );  // Give process identifier
-  }
+  {
+	#if defined(TVCompf_MinGW) && defined(HAVE_64BITS_POINTERS)
+		return( *pidChild= (uint64_t)pi.hProcess );  // Give process identifier
```

Comment:
Without this cast, MingW will not compile this code in 64-bit mode. I don't want to require -fpermissive on the whole project.

```
g++  -I../include        -O2 -Wall -ggdb3  -c ../classes/wingr/wingrscr.cc -o obj/wingrscr.o  

../classes/wingr/wingrscr.cc: In static member function 'static int TScreenWinGr::System(const char*, unsigned int*, int, int, int)':  

../classes/wingr/wingrscr.cc:740:36: error: cast from 'HANDLE' {aka 'void*'} to 'unsigned int' loses precision [-fpermissive]  
  740 |                 return( *pidChild= (pid_t)pi.hProcess );  // Give process identifier  

      |                                    ^~~~~~~~~~~~~~~~~~  

make[1]: *** [librhtv.umk:1441: obj/wingrscr.o] Error 1 
```

Reply:
But with this cast you don't solve the problem. You are just forcing the compilation, if anybody tries to use this value things won't work because you are discarding 32 bits of the 64 bits.

Reply:
Please see latest commit, I believe I've addressed this issue.